index.bs

```diff
@@ -319,6 +320,17 @@ To determine whether an |attribute| and |element| <dfn>attribute-match</dfn> an
 4. if |matches| contains the string "*", return true.
 5. return false.
 
+Some HTML elements require special treatment in a way that can't be easily
+expressed in terms of configuration options or other algorithms. The following
+algorithm collects these in one place. To <dfn>handle funky elements</dfn>,
+run these steps:
+
+1. if |element|'s name is `template`:
+  1. run the sanitizer on the `content` attribute as if `sanitizeToString`
```

Comment:
1. Can we point to our "sanitize a document fragment" algorithm?
2. The [template's content attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template#attributes) is a DocumentFragment. I think we might be able to replace the content attribute with the newly created DocumentFragment. No string serialization required, afaiu.

Reply:
Good idea. Done.